### ISSUE TITLE
AUR "build" doesn't work.

### ISSUE THREAD

**Issue by C8H10N4O2-E970** [nikp123/xava#41]
When I install `xava-git` from AUR. I get the following error.
```
[ontake@ontakePC ~]$ xava
xava: error while loading shared libraries: libxava-shared.so: cannot open shared object file: No such file or directory
```

**Comment by C8H10N4O2-E970** [comment]
Issue is due to `master` not building correctly.

**Comment by nikp123** [comment]
fixed

**Comment by nikp123** [comment]
Provide logs, although it seems that building with yay as opposed to just doing a ```makepkg -sri``` breaks the package

God I hate linker issues.

**Comment by nikp123** [comment]
Also hate to ask this, but could you provide any logs so I can have a deeper look because "it works on my machine".

**Comment by nikp123** [comment]
Except I can't recreate it outside of AUR......

**Comment by nikp123** [comment]
@C8H10N4O2-E970 try now

**Comment by nikp123** [comment]
```
Total Installed Size:  0.64 MiB
Net Upgrade Size:      0.12 MiB
```

XAVA: Now with iniparser included

**Comment by C8H10N4O2-E970** [comment]
Still has the issue

**Comment by C8H10N4O2-E970** [comment]
Manually removing `/usr/local/bin/xava` `/usr/local/lib/xava/` (somehow don't delete with `yay -R xava-git`) and doing `yay -S xava-git` fixes it.

**Comment by nikp123** [comment]
Yeah, I should just disallow /usr/local installs. They cause too much trouble.
